frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -116,19 +151,29 @@ class PopupCreateFile extends React.Component {
     })
   }
 
-  handleValidate = async () => {
-    const { props, state } = this
+  postFile = async (file) => {
+    const { state, props } = this
 
     const formData = new FormData()
-    formData.append('files', state.uploadFile)
+    formData.append('files', file)
     formData.append('parent_id', state.folderId || 0)
+    let filePosted = file
 
-    // fetch still doesn't handle event progress. So we need to use old school xhr object
+    // INFO - CH - 2018-08-28 - fetch still doesn't handle event progress. So we need to use old school xhr object
     const xhr = new XMLHttpRequest()
-    xhr.upload.addEventListener('loadstart', () => this.setState({ progressUpload: { display: false, percent: 0 } }), false)
-    const uploadInProgress = e => e.lengthComputable && this.setState({ progressUpload: { display: true, percent: Math.round(e.loaded / e.total * 100) } })
+
+    const uploadInProgress = e => {
```

Comment:
postFile function can be called multiple times. There is no need to redeclare the function uploadInProgress every time